components/multiselect/multiselect_list.jsx

```diff
@@ -108,12 +108,19 @@ export default class MultiSelectList extends React.Component {
                 className={rowSelected}
                 key={'multiselectoption' + option.value}
                 onClick={() => onAdd(option)}
+                onMouseMove={onMouseMove}
             >
                 {option.label}
             </div>
         );
     }
 
+    onMouseMove = (i) => () => {
```

Comment:
`onMouseMove` being a higher order function threw me for a loop, since I thought it was being used directly as an event handler. I see it getting called on line 159, binding the value of `i`, but at a minimum, I think it might be clearer to just inline the anonymous function creation there. Alternatively, could `onMouseMove` accept the `option` (like the `onAdd` callback does), and use that to know how to configure the `selected` state?

Reply:
I'd lean towards making it work like `onAdd`. It might've been done with a higher order function to avoid the "don't use arrow functions in render" problem, but this has the same root problem

Reply:
@s4kh, sorry for the ambiguity here: this was the only outstanding comment to resolve. Thoughts?